Modules/MUON/MCH/src/PhysicsTaskDigits.cxx

```diff
@@ -186,18 +187,28 @@ void PhysicsTaskDigits::startOfCycle()
 
 void PhysicsTaskDigits::monitorData(o2::framework::ProcessingContext& ctx)
 {
-  // get the input preclusters and associated digits with the orbit information
-  auto digits = ctx.inputs().get<gsl::span<o2::mch::Digit>>("digits");
-  auto orbits = ctx.inputs().get<gsl::span<uint64_t>>("orbits");
-  if (orbits.empty()) {
-    ILOG(Info, Support) << "WARNING: empty orbits vector" << AliceO2::InfoLogger::InfoLogger::endm;
-    return;
+  bool hasOrbits = false;
+  for (auto&& input : ctx.inputs()) {
+    if (input.spec->binding == "orbits") {
+      hasOrbits = true;
+    }
   }
 
-  for (auto& orb : orbits) {
-    storeOrbit(orb);
+  if (hasOrbits) {
+    auto orbits = ctx.inputs().get<gsl::span<uint64_t>>("orbits");
+    if (orbits.empty()) {
+      ILOG(Info, Support) << "WARNING: empty orbits vector" << AliceO2::InfoLogger::InfoLogger::endm;
```

Comment:
In general, it is not recommended to publish log messages in `monitorData` because it can easily flood the infologger in case the condition is true. One way to avoid that is to use a feature of infologger allowing to limit the number of messages per period of time: 
```
    static AliceO2::InfoLogger::InfoLogger::AutoMuteToken msgLimit(LogWarningSupport, 1, 600); // send it once every 10 minutes
    string msg = "object " + path + " is bigger than the maximum allowed size (" + to_string(mMaxObjectSize) + "B) - skipped";
    ILOG_INST.log(msgLimit, "%s", msg.c_str());
```

Reply:
@Barthelemy thanks for the code snippet. While the empty orbits message is an abnormal condition that should never occur, I agree that a protection from messages flood is a good idea.
Should the code go in the `configure()` method of the task?

Reply:
@Barthelemy thanks! I have modified the code as you suggest.